x-pack/legacy/plugins/reporting/public/views/management/index.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
```

Comment:
@mattkime I'm not sure if this is OK or not for our entrypoint. We're still in the shim-phase, so there isn't yet a public `start/stop` calls, just this license subscriber that starts the app.

Reply:
Looks ok to me, as long as it turns out we can move this code to `x-pack/plugins/reporting/public/plugin.tsx` asap, but probably in another PR